### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by roshan-bajgain**
## Vet clinic database: database performance audit
- The following queries are taking too much time (1 sec = 1000ms can be considered as too much time for database query). Try them on your machine to confirm it
- Use EXPLAIN ANALYZE on the previous queries to check what is happening. Take screenshots of them - they will be necessary later
- Find a way to decrease the execution time of the first query. Look for hints in the previous lessons
- Find a way to improve execution time of the other two queries.
- While you are making changes, check if they help by running EXPLAIN ANALYZE. Once you succeed, take a screenshot of the EXPLAIN ANALYZE result showing that time actually decreased.
- Any changes you made to your database schema should be added to the schema.sql file and commited in a new pull request.
- In your pull request description include screenshots with EXPLAIN ANALYZE results (before and after you improve the speed) for each of the 3 problematic queries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Vet clinic database: create animals table
 > In this project I use a relational database to create the initial data structure for a vet clinic. I create a table to store animals' information, insert some data into it, and query it.
-
+## Before
+![Screenshoot-Before](./Before.png)
+## After
+![Screenshoot-After](./After.png)
```
Comment (descholar-ceo) [inline README.md]: - [x] I think you should have included a pair of screenshots with the results of the  EXPLAIN ANALYZE command for each of those queries (in total it should be 6 screenshots)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @roshan-bajgain and @netman5 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (roshan-bajgain) [reply README.md]: Thank you for your suggestion.

**Comment by AshakaE** [comment]
Hi @roshan-bajgain , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**